pkg/sloop/webserver/debug.go

```diff
@@ -123,23 +124,46 @@ func listKeysHandler(tables typed.Tables) http.HandlerFunc {
 		var keys []string
 
 		count := 0
+		totalCount := 0
+		var totalSize int64 = 0
 		err = tables.Db().View(func(txn badgerwrap.Txn) error {
-			keyPrefix := "/" + table + "/"
+			keyPrefix := ""
+			if table != "internal" {
+				keyPrefix = "/" + table + "/"
+			}
+
 			iterOpt := badger.DefaultIteratorOptions
 			iterOpt.Prefix = []byte(keyPrefix)
+			iterOpt.AllVersions = true
+			iterOpt.InternalAccess = true
 			itr := txn.NewIterator(iterOpt)
 			defer itr.Close()
 
-			for itr.Seek([]byte(keyPrefix)); itr.ValidForPrefix([]byte(keyPrefix)); itr.Next() {
+			for itr.Rewind(); itr.Valid(); itr.Next() {
+				totalCount++
 				thisKey := string(itr.Item().Key())
 				if keyRegEx.MatchString(thisKey) {
-					keys = append(keys, thisKey)
-					count += 1
-					if count >= maxRows {
-						glog.Infof("Reached max rows: %v", maxRows)
-						break
+					if table == "internal" {
+						if !strings.HasPrefix(thisKey, "/eventcount") && !strings.HasPrefix(thisKey, "/ressum") && !strings.HasPrefix(thisKey, "/watch") && !strings.HasPrefix(thisKey, "/watchactivity") {
+							keys = append(keys, thisKey)
+							count += 1
+							totalSize += itr.Item().EstimatedSize()
+							if count >= maxRows {
+								glog.Infof("Reached max rows: %v", maxRows)
+								break
+							}
+						}
+					} else {
+						keys = append(keys, thisKey)
```

Comment:
consider moving the `else` duplicated code (keys, count, & totalSize) to above the `if` condition